tests/NuGet.Services.BasicSearchTests/StartupTests.cs

```diff
@@ -0,0 +1,61 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using NuGet.Services.BasicSearchTests.Fixtures;
+using NuGet.Services.BasicSearchTests.Models;
+using Xunit;
+
+namespace NuGet.Services.BasicSearchTests
+{
+    public class StartupTests : IClassFixture<StartupTestsFixture>
+    {
+        private readonly StartupTestsFixture _fixture;
+
+        public StartupTests(StartupTestsFixture fixture)
+        {
+            _fixture = fixture;
+        }
+
+        [Fact]
+        public async Task AutocompleteParameterIsObserved()
+        {
+            // Arrange
+            var app = _fixture.App;
+            string query = "Id:angularjs Id:Antlr Id:WebGrease";
+
+            // Act
+            var withPrereleaseResponse = await app.Client.GetAsync("/query?" + Query(new { q = query, prerelease = true }));
+            var withPrerelease = await withPrereleaseResponse.Content.ReadAsAsync<QueryResult>();
+
+            var withoutPrereleaseResponse = await app.Client.GetAsync("/query?" + Query(new { q = query, prerelease = false }));
+            var withoutPrerelease = await withoutPrereleaseResponse.Content.ReadAsAsync<QueryResult>();
+
+            // Assert
+            Assert.Equal(HttpStatusCode.OK, withPrereleaseResponse.StatusCode);
+            Assert.Equal(HttpStatusCode.OK, withoutPrereleaseResponse.StatusCode);
+
+            Assert.Equal("1.2.0-RC1", withPrerelease.GetPackageVersion("angularjs"));
+            Assert.Equal("3.4.1.9004-pre", withPrerelease.GetPackageVersion("Antlr"));
+            Assert.Equal("1.6.0", withPrerelease.GetPackageVersion("WebGrease"));
+
+            Assert.False(withoutPrerelease.ContainsPackage("angularjs"));
+            Assert.Equal("3.1.3.42154", withoutPrerelease.GetPackageVersion("Antlr"));
+            Assert.Equal("1.6.0", withoutPrerelease.GetPackageVersion("WebGrease"));
+        }
+
+        private static string Query(object obj)
```

Comment:
Since the format of query for search is fixed, can we add a helper class for it?

class SearchQueryBuilder
{
    public static string base; --- static or retrieve from somewhere else 
    public int skip {get;set}
    public bool prerease {get;set}
    public override ToString() ---//  base?query?.....
}